rllib/algorithms/tests/test_algorithm_config.py

```diff
@@ -179,6 +179,7 @@ def test_rl_module_api(self):
             .framework("torch")
             .rollouts(enable_connectors=True)
             .rl_module(_enable_rl_module_api=True)
+            .training(_enable_learner_api=True)
```

Comment:
Had to fix a bunch of test cases that were still using learner=False, but RLModule=True. This is not possible anymore.